tools/cmake/cores/Cortex-A5.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020-2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    list(APPEND common_options
+        "-mthumb-interwork"
+        "-marm"
+        "-mfpu=vfpv3"
+        "-mfloat-abi=softfp"
+        "-mno-unaligned-access"
+        "-mcpu=cortex-a5"
+    )
+elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    list(APPEND common_options
+        "-mfpu=vfpv3"
+        "-mfloat-abi=hard"
+        "-mcpu=cortex-a5"
+    )
+endif()
+
+function(mbed_set_cpu_core_definitions target)
+    target_compile_definitions(${target}
+        INTERFACE
+            __CORTEX_A5
+            ARM_MATH_CA5
+            __FPU_PRESENT
+            __CMSIS_RTOS
+            __EVAL
+            __MBED_CMSIS_RTOS_CA5
```

Comment:
(More of a note to the Mbed team for future reference: at some point we might check if all of them are still used and do a clean-up. They are defined for A9 too, so okay to keep for consistency)

Reply:
This is not required, lets remove it.

The old symbols are not used anymore neither (`__MBED_CMSIS_RTOS_CA9` or `__MBED_CMSIS_RTOS_CM`)